expansion/src/main/java/no/unit/nva/expansion/model/ExpandedResource.java

```diff
@@ -372,16 +373,17 @@ private static ObjectNode transformToJsonLd(Publication publication) throws Json
         var jsonString = objectMapper.writeValueAsString(publication);
         var json = (ObjectNode) objectMapper.readTree(jsonString);
         json.put(ID_FIELD_NAME, extractJsonLdId(publication).toString());
-        json.set(JSON_LD_CONTEXT_FIELD, extractJsonLdContext(publication));
+        json.set(JSON_LD_CONTEXT_FIELD, extractJsonLdContext());
         return json;
     }
 
     private static URI extractJsonLdId(Publication publication) {
         return UriWrapper.fromUri(PUBLICATION_HOST_URI).addChild(publication.getIdentifier().toString()).getUri();
     }
 
-    private static JsonNode extractJsonLdContext(Publication publication) {
-        var jsonContext = publication.getJsonLdContext();
+    private static JsonNode extractJsonLdContext() {
+        var jsonContext = Publication.getJsonLdContext(
+            UriWrapper.fromHost(new Environment().readEnv("API_HOST")).getUri());
```

Comment:
Make "API_HOST" constant?

Reply:
Maybe this already is defined as a constant in some common place?

Reply:
Maybe put it to ExpansionConfig file?